hooks/finding-post-processing/Makefile

```diff
@@ -44,45 +50,66 @@ hook-prefix =
 
 build: | install-deps build-images
 
-test: | unit-tests images-export kind-import deploy deploy-test-deps integration-tests
+test: | unit-tests images-export kind-import deploy-prerequisites deploy deploy-test-deps integration-tests
 
-all: | install-deps unit-tests build-images images-export kind-import deploy deploy-test-deps integration-tests
+all: | clean install-deps unit-tests build-images images-export kind-import deploy-prerequisites deploy deploy-test-deps integration-tests
 
-.PHONY: unit-tests install-deps build-images images-export kind-import deploy deploy-test-deps integration-tests all build test
+.PHONY: unit-tests install-deps build-images images-export kind-import deploy-prerequisites deploy deploy-test-deps integration-tests all build test
 
 unit-tests:
-	cd hook && npx --yes --package jest@$(JEST_VERSION) jest
+	@echo ".: 🧪 Starting unit-tests for hook '$(hook)' with 'jest@$(JEST_VERSION)'."
+	cd hook && npx --yes --package jest@$(JEST_VERSION) jest --ci --colors --coverage .
 
 install-deps:
+	@echo ".: ⚙️ Installing all hook specific dependencies."
 	cd hook && npm ci
 
 build-images:
-	@echo Build With BASE_IMG_TAG: $(BASE_IMG_TAG)
+	@echo ".: ⚙️ Build With BASE_IMG_TAG: '$(BASE_IMG_TAG)'."
 	docker build --build-arg=baseImageTag=$(BASE_IMG_TAG) --build-arg=namespace=$(IMG_NS) -t $(IMG_NS)/$(hook-prefix)$(hook):$(IMG_TAG) -f ./hook/Dockerfile ./hook
 
 images-export:
+	@echo ".: ⚙️ Saving new docker image archive to '$(hook-prefix)$(hook).tar'."
 	docker save $(IMG_NS)/$(hook-prefix)$(hook):$(IMG_TAG) -o $(hook-prefix)$(hook).tar
 
 kind-import:
+	@echo ".: 💾 Importing the image archive '$(hook-prefix)$(hook).tar' to local kind cluster."
 	kind load image-archive ./$(hook-prefix)$(hook).tar
 
+deploy-prerequisites:
+	@echo ".: 💾 Deploying prerequisite HelmChart with the docker tag '$(BASE_IMG_TAG)' into kind."
+	# If not exists create namespace where the tests will be executed
+	kubectl create namespace integration-tests --dry-run=client -o yaml | kubectl apply -f -
+	# If not exists create secureCodeBox operator namespace
+	kubectl create namespace securecodebox-system --dry-run=client -o yaml | kubectl apply -f -
+	helm -n securecodebox-system upgrade --install securecodebox-operator secureCodeBox/operator --wait \
+		# --set="image.repository=docker.io/$(IMG_NS)/operator" \
+		# --set="image.tag=$(BASE_IMG_TAG)" \
+		# --set="lurcher.image.repository=docker.io/$(IMG_NS)/lurcher" \
+		# --set="lurcher.image.tag=$(BASE_IMG_TAG)" \
+
```

Comment:
I am not sure if we should install the operator like this for 3 reasons:
1. You are forced to rebuild for every commit when trying to execute the Makefile locally (because operator and scanner/hook use the same image tag pinned to the last commit)
2. This will probably crash when you run the operator locally
3. Changes in the operator (its values) could affect all Hook Scanner Makefiles in the future

I would propose to outsource this step into the Makefile for the operator